OrchidCore/src/main/java/com/eden/orchid/api/theme/pages/OrchidPage.java

```diff
@@ -128,7 +128,7 @@
     @Description("The importance of this page relative to the rest of the pages on your site. Should be a value " +
             "between 0 and 1."
     )
-    protected float relativePriority;
+    protected float relativePriority = .5f;
```

Comment:
This initial value will actually get overwritten when Orchid actually loads options from the page into the variable. Please also add `@FloatDefault(0.5f)` to make sure it defaults to the correct value.

Reply:
Good point.  Added in 65aa3be.